core/src/test/java/com/onelogin/saml2/test/AllTestsSuite.java

```diff
@@ -1,14 +0,0 @@
-package com.onelogin.saml2.test;
-
-import org.junit.runner.RunWith;
-import org.junit.runners.Suite;
-import org.junit.runners.Suite.SuiteClasses;
-
-import com.onelogin.saml2.test.settings.SettingBuilderTest;
-import com.onelogin.saml2.test.util.UtilsTest;
-
-@RunWith(Suite.class)
-@SuiteClasses({ UtilsTest.class, SettingBuilderTest.class })
```

Comment:
We can use `mvn test` to run the tests, no need to try to keep this file and `ConsoleTestRunner` up to date.